src/directives/uiSrefActive.ts

```diff
@@ -82,6 +82,11 @@ import { Subscription } from 'rxjs';
  */
 @Directive({
   selector: '[uiSrefActive],[uiSrefActiveEq]',
+  hostDirectives: [{
```

Comment:
Would you be able to explain what this change does?

Reply:
Hi @wawyed! Sure, that change is the Directive composition API from the angular framework. This is a new feature, introduced with standalone components, that is intended for directive/components that depend on other directives. It allows you to build features for a component as separated directive that can be reused by others components as well. In this case Angular will create an instance of the `UISrefStatus` directive for each instance of the `UISrefActive`. The selected inputs/outputs from `UISrefStatus`, will become part of the `UISrefActive` API. 

Here is the documentation of the feature https://angular.dev/guide/directives/directive-composition-api